addons/website/models/website_visitor.py

```diff
@@ -31,7 +31,7 @@ class WebsiteVisitor(models.Model):
     _order = 'last_connection_datetime DESC'
 
     name = fields.Char('Name')
-    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, index=True, copy=False, groups='base.group_website_publisher')
+    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, copy=False, groups='base.group_website_publisher')
```

Comment:
I'd keep it explicit to make it obvious that it wasn't just forgotten.

```suggestion
    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, index=False, copy=False, groups='base.group_website_publisher')
```